composer.json

```diff
@@ -21,7 +21,7 @@
     },
     "require-dev": {
         "codeigniter/coding-standard": "^1.1",
-        "codeigniter4/codeigniter4": "dev-develop",
+        "codeigniter4/framework": "^4",
```

Comment:
Please remove `composer.json` changes.
It is not related to this improvement.

> One thing at a time: A pull request should only contain one change.
https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#branching